### ISSUE TITLE
Not performing Validaiton while training

### ISSUE THREAD

**Issue by itsron143** [Samjith888/Keras-retinanet-Training-on-custom-datasets-for-Object-Detection-#7]
Hi,
Thank for this! I am able to train the model but no validation is being performed to check if the model is performing well or not, how can I add that to the code?

**Comment by Samjith888** [comment]
> Hi,
> Thank for this! I am able to train the model but no validation is being performed to check if the model is performing well or not, how

> Hi,
> Thank for this! I am able to train the model but no validation is being performed to check if the model is performing well or not, how can I add that to the code?

You can calculate the mAP value by passing the following line into ur training command ( for this u need a val.csv file)
`--compute-val-loss --val-annotations val.csv`

Also use the tensorboard which is already implemented here( use the tensorboard command in the tensorboard directory)
Then u can see the graph too

**Comment by itsron143** [comment]
Thanks! I'll make changes accordingly to `build_dataset.py` as well to include a 10% split to validaiton as well.
